syftbox/server/server.py

```diff
@@ -166,6 +171,7 @@ async def lifespan(app: FastAPI, settings: Optional[ServerSettings] = None):
 
 
 app = FastAPI(lifespan=lifespan)
+FastAPIInstrumentor.instrument_app(app=app, http_capture_headers_server_request=["email"])
```

Comment:
where is this email coming from?

Reply:
@khoaguin please check the changes i made

Reply:
> where is this email coming from?

![image](https://github.com/user-attachments/assets/3c724f50-2790-4720-94bd-83f701f2eb6a)